### PR TITLE
Text color config login view

### PR DESCRIPTION
**Pull request by malkomich**
New branding for login text color, changing all text but hints and text of AppCompat buttons

Moved 'New to ownCloud' link to bottom of display screen.

### CODE REVIEW

File: res/values/setup.xml [hunk 1]
```diff
@@ -28,6 +28,7 @@
     
     
     <!-- Colors -->
+    <color name="login_text_color">#757575</color>
```
Comment (AndyScherzinger) [inline res/values/setup.xml]: If the login_text_color is moved to setup.xml shouldn't the login_text_hint_color also being moved here, or rather reference login_text_color to make sure it the hint color matched the text color? :)

**Comment by AndyScherzinger** [comment]
Just one minor comment. Besides that: :+1:

**Comment by jabarros** [comment]
LGTM

**Comment by malkomich** [comment]
I have just indent the button tag inside the linearlayout.

The hint text color are not planned to be configurable for the moment @AndyScherzinger, we want to have a minimum number of customizable colors ;)

**Comment by malkomich** [comment]
Well, finally we've decide to add new setup option for hint color, as @AndyScherzinger commented :v:

**Comment by malkomich** [comment]
Ready for a final review

**Comment by AndyScherzinger** [comment]
@malkomich Looks good to me one thing I realized is that you changed account_setup.xml
 but didn't change the corresponding account_setup.xml  in the landscape folder. Can't do the changes at the moment myself, so feel free.

**Comment by malkomich** [comment]
Done, thanks @AndyScherzinger :smiley:

**Comment by AndyScherzinger** [comment]
Nice! :smiley: :+1:

**Comment by davivel** [comment]
Tested and reviewed -> in.

Thanks everybody.

**Pull request merged**